.github/workflows/build-solution.yml

```diff
@@ -15,9 +15,10 @@ on:
         required: true
       PAT:
         required: true
-  on:
-    pull_request_review:
-      types: [ submitted, edited ]
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
```

Comment:
Nitpick: Personally I'm in strong favor of a one-goal-one-PR rule. It makes it much more clear why these changes are there. I agree to what you're trying to do, but doesn't that mean we can also disable the other keys here?

Reply:
I don't fully understand what keys are you talking about here, do you want to remove the `push/pull` rules? I believe in that case CI won't start.